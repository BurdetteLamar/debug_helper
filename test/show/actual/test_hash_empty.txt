---
Hash (size=0 name='test_hash_empty'): {}
