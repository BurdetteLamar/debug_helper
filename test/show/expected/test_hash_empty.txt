---
Hash (message='test_hash_empty'):
  size: 0
  default: 
  default_proc: 
